products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp

```diff
@@ -23,13 +23,13 @@ namespace alarms
         if (duration.has_value()) {
             spawnTimer(duration.value());
         }
-        const auto fadeInSettings = utils::getNumericValue<bool>(
+        const auto fadeSettings = utils::getNumericValue<bool>(
```

Comment:
```suggestion
        const auto fadeEnabledInSettings = utils::getNumericValue<bool>(
```
That way it's more self-explanatory, `fadeSettings` might suggest that this is an object containing multiple settings, not just a simple boolean flag.